src/assets/content/dictionary/automation.md

```diff
@@ -1,6 +1,10 @@
 ---
 title: automation
-definition: ''
-perspectives: []
+definition: Automation is a term for technology applications where human input is minimized. This includes business process automation (BPA), IT automation, personal applications such as home automation and more.
+sources:
+- sourceurl: https://www.ibm.com/topics/automation
+perspectives:
+- meaning: Things Which Minimize Human Effort.
+- role: Developer
```

Comment:
```suggestion
  role: Developer
```
Bug: We don't need a dash in front of the "role" since it will try to create an additional perspective. You can click "Accept" here to accept these changes and I'll approvei t for you.